molecule/vault/tests/test_vault.py

```diff
@@ -0,0 +1,6 @@
+def test_vault(host):
+    cmd = host.run("bash -c 'PS1=yes ; source /root/.bashrc ; vault status'")
+    print(cmd.stdout)
+    print(cmd.stderr)
+    assert 'Initialized     true' in cmd.stdout
```

Comment:
`vault status -format=json` could be used (parsing `json` output is easier than parsing default `table` format).

Reply:
I will do that in [another, larger, pull request](https://github.com/open-craft/ansible-playbooks/pull/99/files#diff-1be18bdbd3a11bbc7711b8c27df13bdaR7)